recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
```

Comment:
Maybe we should add a check for [`TRAVIS`]( https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables ) too.